source/common/dmtbinfo2.c

```diff
@@ -924,9 +922,88 @@ ACPI_DMTABLE_INFO           AcpiDmTableInfoMadt16[] =
    ACPI_DMT_TERMINATOR
 };
 
-/* 17: OEM data structure */
+/* 17: core interrupt controller */
 
 ACPI_DMTABLE_INFO           AcpiDmTableInfoMadt17[] =
+{
+    {ACPI_DMT_UINT8,    ACPI_MADT17_OFFSET (Version),            "Version", 0},
```

Comment:
Strings are de-indented by 2 relative to all the previous MADT subtables, later become de-indented by only 1 and eventually for table 23 they're properly indented again

Reply:
Same above, thanks.